### PR TITLE
Ram: FW version dump

### PR DESCRIPTION
**Pull request by realfast**
**Description** [Add in extra firmwares from Ram DT and Ram HD.]

**Verification** [These were brought in from various trucks running my fork.]

### CODE REVIEW

**Comment by sshane** [comment]
Can you provide some dongle IDs for where these came from?

**Comment by realfast** [comment]
Below are a few. I will have to really dig to find all of these

348699f3dc65b1d2,
1ea6ef1d853f9a07,
665a8c22095fb213,
8fcf23b44e784fe4,
5b90d365660e4054,
8ef49af09e17329e,
2a26efe7d0cdb4c5,
6c9f3091a4c035d8,
ebf413d369473f7a,
5c4ebbe8a5347b6d,
4f575700ecddf660,
e98b5d809abaf860,

File: selfdrive/car/chrysler/values.py [hunk 1]
```diff
@@ -181,85 +181,143 @@ class ChryslerCarInfo(CarInfo):
...
       b'68475160AE',
+      b'68475160AF',
+      b'68475160AG',
     ],
     (Ecu.eps, 0x75A, None): [
-      b'68273275AG',
+      b'68273275AF', #Steer to 0 Capable
+      b'68273275AG', #Steer to 0 Capable
+      b'68312176AE', #Steer to 0 Capable
+      b'68312176AG', #Steer to 0 Capable
```
Comment (sshane) [inline selfdrive/car/chrysler/values.py:240]:
```suggestion
      b'68273275AF',
      b'68273275AG',
      b'68312176AE',
      b'68312176AG',
```

File: selfdrive/car/chrysler/values.py [hunk 2]
```diff
@@ -182,85 +182,143 @@ class ChryslerCarInfo(CarInfo):
...
       b'68361606AH',
+      b'68437735AC',
+      b'68437746AD',
       b'68492693AD',
+      b'68525485AB',
+      b'68525487AB',
+      b'68525498AB',
     ],
     (Ecu.srs, 0x744, None): [
+      b'68346749AB',
```
Comment (sshane) [inline selfdrive/car/chrysler/values.py:288]:
```suggestion
```

I saw no valid routes with this FW in the last year, so removing for this PR

File: selfdrive/car/chrysler/values.py [hunk 3]
```diff
@@ -182,85 +182,143 @@ class ChryslerCarInfo(CarInfo):
...
+      b'68502994AD',
+      b'68540431AB',
     ],
   },
 
   CAR.RAM_HD: {
     (Ecu.combinationMeter, 0x742, None): [
       b'68361606AH',
+      b'68437735AC',
+      b'68437746AD',
```
Comment (sshane) [inline selfdrive/car/chrysler/values.py:281]:
```suggestion
```

same with this

File: selfdrive/car/chrysler/values.py [hunk 4]
```diff
@@ -182,85 +182,143 @@ class ChryslerCarInfo(CarInfo):
...
     (Ecu.transmission, 0x7e1, None): [
+      b'05036069AA',
       b'68360078AL',
-      b'68384328AD',
-      b'68360085AL',
+      b'68360080AM',
       b'68360081AM',
-      b'68502994AD',
+      b'68360085AL',
+      b'68360085AM',
```
Comment (sshane) [inline selfdrive/car/chrysler/values.py:267]:
```suggestion
```

File: selfdrive/car/chrysler/values.py [hunk 5]
```diff
@@ -182,85 +182,143 @@ class ChryslerCarInfo(CarInfo):
...
+      b'68378758AM ',
       b'68448163AJ',
+      b'68448165AK',
       b'68500630AD',
+      b'68500630AE',
       b'68539650AD',
-      b'68378758AM ',
     ],
     (Ecu.transmission, 0x7e1, None): [
+      b'05036069AA',
```
Comment (sshane) [inline selfdrive/car/chrysler/values.py:262]:
```suggestion
```

**Comment by sshane** [comment]
Checked the VINs for the dongles you added, found we have a 2023! Pulled out into this PR: https://github.com/commaai/openpilot/pull/27634

File: selfdrive/car/chrysler/values.py [hunk 6]
```diff
@@ -182,83 +182,126 @@ class ChryslerCarInfo(CarInfo):
...
+      b'68434858AC',
       b'68434860AC',
-      b'68527375AD',
       b'68453503AC',
+      b'68453505AC',
+      b'68453511AC',
+      b'68453513AD',
+      b'68453514AD',
       b'68510283AG',
+      b'68527375AD',
```
Comment (sshane) [inline selfdrive/car/chrysler/values.py:200]: These look good! They all come from 2019-21 cars

**Comment by sshane** [comment]
I parsed the VIN for all the FW versions you added, and they all check out. Removed all dongles and VINs for privacy: https://pastebin.com/1yWJmfs0

**Pull request merged**